src/main/kotlin/no/nav/familie/ef/sak/brev/FritekstBrevController.kt

```diff
@@ -10,13 +9,13 @@ import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
-@RequestMapping(path = ["/api/manueltbrev"])
+@RequestMapping(path = ["/api/brev"])
```

Comment:
Denne base-path er samme som i `VedtaksbrevController`, og da kan vi kanksje få kollisjoner i fremtiden. Foreslår vi endrer denne til `api/frittstaende-brev`
```suggestion
@RequestMapping(path = ["/api/frittstaende-brev"])
```

Reply:
Fikset i 2d7cca8